Makefile

```diff
@@ -34,12 +34,14 @@ pandoc:
 	echo "" >> pandoc.TMP
 	cat previous-exp/EDR-at-KNMI.md >> pandoc.TMP
 	echo "" >> pandoc.TMP
+	cat previous-exp/current-data-management.md >> pandoc.TMP
```

Comment:
does this file exist? https://github.com/EURODEO/e-soh-poc-report/tree/main/previous-exp